src/main/kotlin/frc/robot/Constants.kt

```diff
@@ -22,6 +22,10 @@ val CURRENT_MODE: Mode
                 Mode.SIM
             }
         }
+
+val USE_MAPLE_SIM: Boolean
+    get() = System.getenv()["isMapleSim"] == "true"
```

Comment:
Copying from line 32:

```suggestion
    get() = System.getenv("isMapleSim") == "true"
```

Reply:
In that case `System.getenv()["isReplay"] == "true"` should also change for consistency.
dd7a6e883032aecd8def342c9c701009cd293219

Reply:
Awesome